src/dotnet/APIView/APIViewWeb/Pages/Assemblies/Review.cshtml.cs

```diff
@@ -38,14 +43,23 @@ public async Task OnGetAsync(string id)
             var assemblyModel = await assemblyRepository.ReadAssemblyContentAsync(id);
             var renderer = new HTMLRendererAPIV();
             AssemblyModel = renderer.Render(assemblyModel.Assembly);
+            var comments = await commentRepository.FetchCommentsAsync(id);
 
-            Comments = await commentRepository.FetchCommentsAsync(id);
+            Comments = new Dictionary<string, List<CommentModel>>();
+            _ = new List<CommentModel>();
```

Comment:
Hm?

Reply:
That guy on line 49 shouldn't exist anymore - went ahead and removed it